CITATION.cff

```diff
@@ -0,0 +1,31 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
```

Comment:
let's remove this

Reply:
okay.

Reply:
I kept it since felt it's more ethic to cite/quote their work :)